src/actions/index.js

```diff
@@ -84,7 +86,25 @@ export const getSchoolsFailure = () => ({
   type: actionTypes.GET_SCHOOLS_FAILURE,
 });
 
+//////
+export function FETCH_SUBJECTS_FAILURE(){
```

Comment:
You don't need this (duplicate of action above)

Reply:
Ah I messed up resolving the merge conflicts. I see.